stories/covid-vax/covid_vax.ipynb

```diff
@@ -0,0 +1,1597 @@
+{
```

Comment:
<p>I may be reading too much into a single word, but would it be better to say "Step 2: Defining which data we wish to retrieve"? Statistics implied to me some analysis has already been done on these numbers, but they are just a vaccine dose counts. Full disclosure: I'm a social scientists, not a data scientist, so I may be totally wrong on this terminology.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/alan-turing-institute/TuringDataStories/pull/138/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
<p>"Data"  is clear and makes the sentence shorter and snappier so let's go with that! </p>